src/app/shared/faq-assistant/faq-assistant.component.scss

```diff
@@ -30,18 +34,32 @@
 
 .component-style__content {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   flex-wrap: wrap;
   justify-content: center;
-  padding: 30px;
-  box-shadow: 0px 0px 4px -1px;
+  overflow-y: scroll;
+  height: 300px;
+  width: 320px;
+  padding: 30px 15px 100px 15px;
+  box-shadow: 0 0 4px -1px;
+  background: whitesmoke;
```

Comment:
It will be good if we move colors to vars.

Reply:
was applied